packages/interfaces/src/message.ts

```diff
@@ -69,6 +69,21 @@ export interface IMessage {
   rateLimitProof?: IRateLimitProof;
 }
 
+/**
+ * Send message data structure used in {@link IWaku.send}.
+ */
+export interface ISendMessage {
+  contentTopic: string;
+  payload: Uint8Array;
+  ephemeral?: boolean;
+  rateLimitProof?: boolean;
```

Comment:
I don't think the API consumer should be exposed to the rate limit proof, this sit behind the Waku API. see  `createNode` spec for details.